src/bin/cratesfyi.rs

```diff
@@ -454,6 +461,16 @@ impl DatabaseSubcommand {
                     .update_all_crates()?;
             }
 
+            Self::UpdateCrateRegistryFields { name } => {
+                let index = Index::new(&ctx.config()?.registry_index_path)?;
+
+                db::update_crate_data_in_database(
+                    &*ctx.conn()?,
+                    &name,
+                    &index.api().get_crate_data(&name),
+                )?;
```

Comment:
So, this is ok for now because it's just one or two crates. But when we automate this for the full consistency check, it should use the database dump to avoid spamming the crates.io API. @pietroalbini will not be happy if we make 300k requests in a day 😆